proto/rill/runtime/v1/queries.proto

```diff
@@ -389,6 +389,7 @@ message MetricsViewAggregationMeasure {
     MetricsViewAggregationMeasureComputeComparisonValue comparison_value = 7;
     MetricsViewAggregationMeasureComputeComparisonDelta comparison_delta = 8;
     MetricsViewAggregationMeasureComputeComparisonRatio comparison_ratio = 9;
+    MetricsViewAggregationMeasurePercentOfTotal percent_of_total = 10;
```

Comment:
nit: put "compute" in the name like for the other computed measure types: `MetricsViewAggregationMeasureComputePercentOfTotal`